### ISSUE TITLE
Unable to install package pyeasee==0.7.50: WARNING: Ignoring invalid distribution -sgpack

### ISSUE THREAD

**Issue by emilhauk** [fondberg/pyeasee#82]
I'm attempting to install https://github.com/fondberg/easee_hass into my Home Assistant installation running on official docker image https://github.com/home-assistant/core/pkgs/container/home-assistant/90447435?tag=stable. This has been an issue for several releases of this image, so it's not new behaviour on this latest release. Running on x86_64 platform.

Relevant error from logs:
```
2023-05-04 21:31:15.772 ERROR (SyncWorker_0) [homeassistant.util.package] Unable to install package pyeasee==0.7.50: ERROR: Could not install packages due to an OSError: [Errno 22] Invalid argument: '/usr/local/lib/python3.10/site-packages/msgpack-1.0.5.dist-info/'
``` 

Any suggestions on how to proceed? (Not a native Python writer :) )

**Comment by olalid** [comment]
I think your installation is somehow corrupted.
If you delete the folder /usr/local/lib/python3.10/site-packages/msgpack-1.0.5.dist-info/ and /usr/local/lib/python3.10/site-packages/msgpack/ and restart HA, does that make any difference?
You can also try deleting /usr/local/lib/python3.10/site-packages/signalrcore-0.9.5.dist-info/ and /usr/local/lib/python3.10/site-packages/signalrcore/ if that does not help.

**Comment by emilhauk** [comment]
After some more testing it seems to come down to how i run home assistant.

I run docker inside an LXC-container on Proxmox. I just set up a basic compose-file. Ran it locally and it installed just fine. Then I ran the same compose-file inside the LXC-container and it failed in the same way as my primary install.

My primary install has other custom components that install additional libraries, which installs and run without issues.

Further investigation is required.
